java/ql/src/Security/CWE/CWE-652/XQueryInjectionLib.qll

```diff
@@ -0,0 +1,35 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+class XQueryInjectionConfig extends TaintTracking::Configuration {
```

Comment:
Doc comment for this and all other public elements please (i.e., also `XQueryInjectionSink`)